index.html

```diff
@@ -0,0 +1,49 @@
+<html>
+<head>
+<link rel="stylesheet" href="styles.css">
+</head>
+<body>
+<div id="main">
+<div id="title-div">
```

Comment:
I think you should remove these <div> tags because there is no need for these tags.

Reply:
- [ ] Good point mentioned, please give more details on why there is no need for `div` tags. giving more details let your partner understand and realize the bugs easily.

Reply:
I have mentioned that <div> tags should only be used when they have any use. Using any extra <div> tag without any use would take some space and would disturb the cleanliness of the code.